app/forms/steps/capital/property_form.rb

```diff
@@ -19,6 +19,13 @@ class PropertyForm < Steps::BaseFormObject
       validates :has_other_owners, inclusion: { in: YesNoAnswer.values }
       validates :percentage_partner_owned, presence: true, if: :include_partner?
 
+      validates_numericality_of :percentage_applicant_owned, greater_than_or_equal_to: 1.0,
+                                less_than_or_equal_to: 100.0
+      validates_numericality_of :percentage_partner_owned, greater_than_or_equal_to: 1.0,
+                                less_than_or_equal_to: 100.0, if: :include_partner?
```

Comment:
I'm wondering if this is too restrictive.. I'm assuming the client would always have to own or part own the property but the partner could be 0?